src/components/Card.ts

```diff
@@ -1,57 +1,55 @@
 import styled, { css } from 'styled-components';
 import { media } from 'components/breakpoints';
+import { GU } from 'components/theme';
 
 export const Card = styled.div`
-    backdrop-filter: blur(10px);
-    background: rgba(255, 255, 255, 0.6);
-    border: 3px solid #000000;
-    box-sizing: border-box;
-    margin: 0 auto;
-    min-height: 10rem;
-    width: 30rem;
+  backdrop-filter: blur(${3 * GU}px);
+  background: rgba(255, 255, 255, 0.6);
+  border: ${GU}px solid #000000;
+  box-sizing: border-box;
+  margin: 0 auto;
+  min-height: ${25 * GU};
+  width: ${80 * GU}px;
 
-    ${media.xs`
-        border: 5px solid #000000;
-        width: 48rem;
+  ${media.xs`
+        width: ${120 * GU}px;
     `}
 
-    ${media.sm`
-        width: 76rem;
+  ${media.sm`
+        width: ${190 * GU}px;
     `}
```

Comment:
All nested responsive queries like these seem to be indented wrong—The backtick should match the dollar sign and then everything else inside should be indented two spaces. Let's fix 'em up before merging!

Reply:
Fixed in 080bd6ffc108c5a8360f6c2eced87e166f86de18